libs/labelbox/src/labelbox/schema/send_to_annotate_params.py

```diff
@@ -29,14 +31,23 @@ class SendToAnnotateFromCatalogParams(TypedDict):
         ConflictResolutionStrategy.KEEP_EXISTING.
     :param batch_priority: Optional[int] - The priority of the batch. Defaults to 5.
     """
-
-    source_model_run_id: Optional[str]
-    predictions_ontology_mapping: Optional[Dict[str, str]]
-    source_project_id: Optional[str]
-    annotations_ontology_mapping: Optional[Dict[str, str]]
-    exclude_data_rows_in_project: Optional[bool]
-    override_existing_annotations_rule: Optional[ConflictResolutionStrategy]
-    batch_priority: Optional[int]
+    model_config = ConfigDict(extra='forbid')
+
+    source_model_run_id: Optional[str] = None
+    source_project_id: Optional[str] = None
+    predictions_ontology_mapping: Optional[Dict[str, str]] = {}
+    annotations_ontology_mapping: Optional[Dict[str, str]] = {}
+    exclude_data_rows_in_project: Optional[bool] = False
+    override_existing_annotations_rule: Optional[
+        ConflictResolutionStrategy] = ConflictResolutionStrategy.KeepExisting
+    batch_priority: Optional[int] = 5
+
+    @model_validator(mode='after')
+    def check_project_id_or_model_run_id(self):
+        if not self.source_model_run_id and not self.source_project_id:
+            raise ValueError(
+                'source_project_id or source_model_id are required inside params dictionary'
```

Comment:
I would just say `Either source_project_id or source_model_id are required`

Reply:
Is there a check that both should not be present?

Reply:
Added